crates/notion-core/src/tool/mod.rs

```diff
@@ -123,6 +125,35 @@ impl ToolSpec {
             package => ToolSpec::Package(package.into(), version),
         })
     }
+
+    /// Check the args for the bad pattern of `notion install <tool> <number>`.
```

Comment:
What about `notion <action> <tool> <number> <tool>…`?  E.g. should we also allow `notion install node 10 yarn ember-cli`?

Reply:
Good point – @charlespierce caught the same.